### DISCUSSION TITLE
Please don't add ADs to GitHub UI

### DISCUSSION THREAD

**Discussion by verdverm** [community/community#65245] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Noticed the following when browsing files, please don't...

https://imgur.com/a/wJY0698

![image](https://github.com/community/community/assets/1390600/1fb89cb5-dcd1-4a48-8e23-b0478c12c40b)

**Comment by crvdgc** [comment]
The other day, I spotted this when browsing a public repo and clicked "don't show this to me again". Apparently this means nothing, and GitHub starts to add "Your Organization can pay for GitHub Copilot" in a paying organization's private repo.

Code browser is the last place that I want any distraction. Is there no shame to whoever proposed this? Or the AI fad has gained so much attraction that supporting normal software development is simply not a priority anymore?

**Reply by ghost** [reply]
As someone with attention issues: I feel this impacts the site's accessibility. I am now using an adblocker for github.

**Reply by verdverm** [reply]
I use uBlock and you can block specific elements as well. Super useful across a great many sites

**Comment by lpcvoid** [comment]
Microslop now calls this a "tip", it's not an ad, nothing to see here, move along!

https://news.ycombinator.com/item?id=47573233
